RadarSDK.xcodeproj/project.pbxproj

```diff
@@ -729,9 +730,10 @@
 		0107A9ED2621FFB9008AB52F /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 96GHH65B9D;
```

Comment:
@jrtibbetts Is this the Radar team ID? Does it matter?

Reply:
👀